runelite-client/src/main/java/net/runelite/client/plugins/inventorygrid/InventoryGridConfig.java

```diff
@@ -35,37 +36,52 @@
 	@ConfigItem(
 		keyName = "showItem",
 		name = "Show item",
-		description = "Show a preview of the item in the new slot"
+		description = "Show a preview of the item in the new slot",
+		position = 0
 	)
 	default boolean showItem()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showGrid",
-		name = "Show grid",
-		description = "Show a grid on the inventory while dragging"
+		keyName = "showHighlight",
+		name = "Highlight background",
+		description = "Show a green background highlight on the new slot",
+		position = 1
 	)
-	default boolean showGrid()
+	default boolean showHighlight()
 	{
 		return true;
 	}
 
 	@ConfigItem(
-		keyName = "showHighlight",
-		name = "Highlight background",
-		description = "Show a green background highlight on the new slot"
+			keyName = "showInventoryGrid",
+			name = "Show inventory grid",
+			description = "Show a grid on the inventory while dragging",
+			position = 2
 	)
-	default boolean showHighlight()
+	default boolean showInventoryGrid()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+			keyName = "showBankGrid",
+			name = "Show bank grid",
+			description = "Show a grid on the bank while dragging",
+			position = 3
+	)
+	default boolean showBankGrid()
 	{
 		return true;
 	}
 
 	@ConfigItem(
 		keyName = "dragDelay",
 		name = "Drag Delay",
-		description = "Time in ms to wait after item press before showing grid"
+		description = "Time in ms to wait after item press before showing grid",
+		position = 4
```

Comment:
Why were positions added to this config?

Reply:
Because the ordering of the was not very intuitive.

Reply:
it sorts by name

Reply:
This maybe true but I looks and makes more sense keeping like options near one another.

Reply:
or don't change their keys and names

Reply:
Removed positions